spec/index.bs

```diff
@@ -4941,8 +4941,8 @@ dictionary GPUPrimitiveState {
     GPUFrontFace frontFace = "ccw";
     GPUCullMode cullMode = "none";
 
-    // Enable depth clamping (requires "depth-clamping" feature)
-    boolean clampDepth = false;
+    // Requires "disable-depth-clip" feature.
+    boolean disableDepthClip = false;
```

Comment:
and then here we could use a proper `enum DepthClipMode { "clip", "clamp" }`

Reply:
The problem with that naming is that's not what it does, even though that's the wording Metal uses for the same thing. Clamping is always enabled; this flag only changes whether clipping is enabled.

Reply:
The underlying API to emulate here is D3D12, which calls it DepthClipEnable which defaults to true.

Reply:
I find "Clamping is always enabled" statement confusing here. Clamping isn't observable if the clipping is ON.

Reply:
Clamping occurs after the fragment shader so it is observable if the fragment shader outputs frag_depth.

AFAICT from testing and specs/docs, clamping never occurs at another point in the pipeline - this was what took me so long to figure out

Reply:
editors `unclippedDepth`?